backup/backup.go

```diff
@@ -11,10 +11,10 @@ import (
 	"sync"
 	"time"
 
-	"github.com/greenplum-db/gp-common-go-libs/cluster"
-	"github.com/greenplum-db/gp-common-go-libs/dbconn"
-	"github.com/greenplum-db/gp-common-go-libs/gplog"
-	"github.com/greenplum-db/gp-common-go-libs/operating"
+	"github.com/cloudberrydb/gp-common-go-libs/cluster"
```

Comment:
```suggestion
	"github.com/apache/cloudberry-go-libs/cluster"
```

Reply:
The branch `apache:sync-with-upstream` should be renamed to main now. Can take a look - https://github.com/apache/cloudberry-go-libs/branches/all. Just FYI.